workflows/ampliseq.nf

```diff
@@ -132,9 +132,11 @@ include { DADA2_RMCHIMERA               } from '../modules/local/dada2_rmchimera
 include { DADA2_STATS                   } from '../modules/local/dada2_stats'                  addParams( options: modules['dada2_stats']          )
 include { DADA2_MERGE                   } from '../modules/local/dada2_merge'                  addParams( options: modules['dada2_merge']          )
 include { FORMAT_TAXONOMY               } from '../modules/local/format_taxonomy'
+include { ITSX_CUTASV                   } from '../modules/local/itsx_cutasv'
```

Comment:
Would be good to add here e.g. `addParams( options: modules['itsx_cutasv'] )` for static params so that those can be modified if needed (but not recommended ofc).